be/src/common/daemon.cpp

```diff
@@ -52,6 +52,7 @@
 #include "util/timezone_utils.h"
 
 namespace starrocks {
+DEFINE_bool(cn, false, "start as compute node");
```

Comment:
I'd prefer `--node-type cn` or `--node-type be` to decide which role this process will be.

Reply:
That will make it too difficult to check it in starrocks_main.cpp before gflags work